prob/torus_cbc/build.py

```diff
@@ -115,7 +116,8 @@
 if FORTRAN:
     OPACPATH = "opacity.SFHo.nohoro.juo.brem1.bin"
 else:
-    OPACPATH = "opacity.SFHo.nohoro.juo.brem1.h5"
+    #OPACPATH = "opacity.SFHo.nohoro.juo.brem1.h5"
+    OPACPATH = "NuLib_rho70_temp62_ye50_ng61_ns4_version1.0_20241120_bhlight.h5"
```

Comment:
What about subjecting this to an 'if oscillations'-type condition? If oscillations, NuLib, else, SFHo?

Reply:
good idea will change that